config/routes.rb

```diff
@@ -12,6 +12,10 @@
   end
 
   resources :statuses, except: [:show] do
-    get :today, on: :collection
+    collection do
+      get :today
+      get :yesterday
+      get '/:year/:month/:date', action: :on # 'statuses/2015', 'statuses/2015/01', 'statuses/2015/01/01'
```

Comment:
Line is too long. [105/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.